lib/Jsrt/Jsrt.cpp

```diff
@@ -889,6 +901,35 @@ CHAKRA_API JsStringToPointer(_In_ JsValueRef stringValue, _Outptr_result_buffer_
     });
 }
 
+CHAKRA_API JsStringToPointerUtf8(_In_ JsValueRef stringValue, _Outptr_result_buffer_(*stringLength) const char **stringPtr, _Out_ size_t *stringLength)
+{
+    const wchar_t* wstr;
+    size_t wstrLen;
+    JsErrorCode err = JsStringToPointer(stringValue, &wstr, &wstrLen);
+    if (err == JsNoError)
+    {
+        PARAM_NOT_NULL(stringPtr);
+        PARAM_NOT_NULL(stringLength);
+        *stringPtr = nullptr;
+        *stringLength = 0;
+
+        // xplat-todo: fix lifetime. caller allocate space, or caller free?
```

Comment:
As of now this leaks right? Can we say `JsStringToPointerUtf8Copy` and caller is responsible from freeing it? but that would be adding a new interface..